cmd/generate/config/rules/atlassian.go

```diff
@@ -11,15 +11,23 @@ func Atlassian() *config.Rule {
 	r := config.Rule{
 		Description: "Detected an Atlassian API token, posing a threat to project management and collaboration tool security and data confidentiality.",
 		RuleID:      "atlassian-api-token",
-		Regex: utils.GenerateSemiGenericRegex([]string{
-			"atlassian", "confluence", "jira"}, utils.AlphaNumeric("24"), true),
+		Regex: utils.MergeRegexps(
+			utils.GenerateSemiGenericRegex(
+				[]string{"atlassian", "confluence", "jira"},
+				utils.AlphaNumeric("24"),
+				true,
+			),
+			utils.GenerateUniqueTokenRegex(`ATATT3[A-Za-z0-9_\-=]{186}`, false),
+		),
+		Entropy:  3.5,
 		Keywords: []string{"atlassian", "confluence", "jira"},
```

Comment:
I think the newer token is unique enough that we can add the prefix to the list of keywords.
```suggestion
		Keywords: []string{"atlassian", "confluence", "jira", "atatt3"},
```